AgentDiscoveries-Frontend/app/src/components/home.jsx

```diff
@@ -27,39 +31,38 @@ export default class Home extends React.Component {
     render() {
         return (
             <div>
-                <center>
-
-                    <h3>{this.state.user ? `Welcome ${this.state.user.username}` : '' }</h3>
-                    <p>This is the CIA's secret website... create secret messages and add reports!!!!!</p>
-
-                    <table width="30%">
-                        <tr>
-                            <th>Location:</th>
-                            <th>Time:</th>
-                        </tr>
-                        <tr>
-                            <td>Africa/Bamako</td>
-                            <td><Clock format={'HH:mm:ss'} ticking={true} timezone={'Africa/Bamako'} /></td>
-                        </tr>
-                        <tr>
-                            <td>America/Chihuahua</td>
-                            <td><Clock format={'HH:mm:ss'} ticking={true} timezone={'America/Chihuahua'} /></td>
-                        </tr>
+                <Table className = "align-properly">
+                    <thead>
                         <tr>
-                            <td>America/New_York</td>
-                            <td><Clock format={'HH:mm:ss'} ticking={true} timezone={'America/New_York'} /></td>
+                            <th className = "align-properly" className = "motto"><h1>{this.state.user ? `Welcome, ${this.state.user.username}` : '' }</h1></th>
```

Comment:
We should be able to attach these two className properties in the same string, just separated by a space